daemon/analytics.c

```diff
@@ -511,13 +511,13 @@ void analytics_misc(void)
 {
 #ifdef ENABLE_ACLK
     analytics_set_data(&analytics_data.netdata_host_cloud_available, "true");
-    if (aclk_ng)
-        analytics_set_data_str(&analytics_data.netdata_host_aclk_implementation, "Next Generation");
-    else
-        analytics_set_data_str(&analytics_data.netdata_host_aclk_implementation, "legacy");
+#ifdef ACLK_NG
+    analytics_set_data_str(&analytics_data.netdata_host_aclk_implementation, "Next Generation");
+#else
+    analytics_set_data_str(&analytics_data.netdata_host_aclk_implementation, "legacy");
```

Comment:
Why `Next Generation` is in upper case while `legacy` is in lowercase?

Reply:
It was like that I think in web/api/info .... And so I left them the same, to be consistent with what we show there and in posthog. Now they are embedded probably in sql queries, so I'll leave them like that. :-)

Reply:
Btw, this part has been moved to https://github.com/netdata/netdata/pull/11247 for a faster merge :-)